Purchases/Networking/Caching/CallbackCache.swift

```diff
@@ -45,17 +45,20 @@ class CallbackCache<T> where T: CacheKeyProviding {
 
     func performOnAllItemsAndRemoveFromCache(withCacheable cacheable: CacheKeyProviding, _ block: (T) -> Void) {
         callbackQueue.sync {
-            guard let items = cachedCallbacksByKey[cacheable.cacheKey] else {
+            guard let items = cachedCallbacksByKey.removeValue(forKey: cacheable.cacheKey) else {
```

Comment:
Oh this is even better 👍

Reply:
yeah, I totally forgot about this func until I was experimenting with some other things.